Classes/YLTableViewDataSource.h

```diff
@@ -16,8 +16,14 @@ NS_ASSUME_NONNULL_BEGIN
 //! If one of model's properties has changed (but it's still the same object), this will reload the visible cell for that model. Useful whenever something mutates the state of a model (i.e. image loading)
 - (void)reloadVisibleCellForModel:(id)model inTableView:(UITableView *)tableView;
 
+//! If the cells implement the YLTableViewCellEstimatedRowHeight protocol, that height will be used. Otherwise, the tableView can use an alternate height specified in this method.
+- (CGFloat)estimatedHeightForRow;
+
 //! Set as the parent view controller of any cells implementing YLTableViewChildViewControllerCell.
 @property (weak, nonatomic, nullable) UIViewController *parentViewController;
 
+//! Defaults to NO. If you would like to cache the estimated heights for rows, set this to YES.
+@property (assign, nonatomic) BOOL shouldCacheEstimatedHeights;
```

Comment:
nit: shouldCacheEstimateRowHeights. Also, I think it's worth explaining that this is only something you would need for iOS 8.